hooks/pre/40-el8_modules.rb

```diff
@@ -0,0 +1,7 @@
+if (module_enabled?('katello') || module_enabled?('foreman_proxy_content')) && el8?
+  ensure_dnf_module('pki-core', 'present')
```

Comment:
I started with https://github.com/theforeman/puppet-candlepin/pull/175. We've observed that starting Puppet typically adds a few seconds so I'd prefer to solve it as part of the main loop.

Reply:
This blocks Katello so we need to get any changes and release ASAP.